### PR TITLE
docs: Fix example to support find node on windows

### LINKED ISSUE #31: find-node example doesn't work on windows
Hey there, I'm having trouble running the example `find-node` step from the README.

I'm getting the following error in the logs for that step:

```
Run echo "node=$(node -e 'console.log(process.argv[0]);')" >> "$GitHub_OUTPUT"
  echo "node=$(node -e 'console.log(process.argv[0]);')" >> "$GitHub_OUTPUT"
  shell: /usr/bin/bash -e {0}
/home/runner/work/_temp/cd[2](https://github.com/[redacted]/actions/runs/10149681769/job/28065132487#step:6:2)d0e4b-1e43-4e77-bc5b-3fd9849dbb59.sh: line 1: : No such file or directory
Error: Process completed with exit code 1.
```

Is it maybe an issue with the "$GITHUB_OUTPUT" variable on windows?

### PR DESCRIPTION
**Pull request by RangerMauve**
Fixes #31

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -48,7 +48,7 @@ sea-action:
       id: find-node
       run:
         echo "node=$(node -e 'console.log(process.argv[0]);')" >>
-        "$GitHub_OUTPUT"
+        $env:GITHUB_OUTPUT
```
Comment (bryopsida) [inline README.md:51]: Do you know if this works on all the runners or just windows?

The reason I ask is GitHub's documentation has it in the form of `>> "$GITHUB_OUTPUT"`.

https://docs.github.com/en/actions/writing-workflows/choosing-what-your-workflow-does/defining-outputs-for-jobs#example-defining-outputs-for-a-job

The CI job uses the same and it looks like it's working, the CI job does use correct casing though `GitHub` -> `GITHUB`

image

 https://github.com/bryopsida/node-sea-action/blob/main/.github/workflows/ci.yml#L78

Comment (RangerMauve) [reply README.md:51]: I tested it on macos/linux/windows on x86 and arm64 :)

Comment (RangerMauve) [reply README.md:51]: I can try the proper casing next week to see if that's enough in case this PR seems to risky.

Comment (bryopsida) [reply README.md:51]: Just wanted to be sure it would work for all runners.

**Pull request merged**